crates/database/src/fee_policies.rs

```diff
@@ -0,0 +1,119 @@
+use {
```

Comment:
Why should this live in the shared database crate vs. autopilot::database?

Reply:
Is it expected that the data will be fetched from the orderbook later?